lib/classes/PluginManager.js

```diff
@@ -41,9 +47,12 @@ class PluginManager {
     }
 
     forEach(options, (value, key) => {
-      if (value.required && (this.options[key] === true || !(this.options[key]))) {
-        const errorMessage = `This command requires the --${key} option. Please pass
-     it through. :)`;
+      if (value.required && (this.cliOptions[key] === true || !(this.cliOptions[key]))) {
+        let requiredThings = `the --${key} option`;
+        if (value.shortcut) {
+          requiredThings += ` / -${value.shortcut} shortcut`;
+        }
+        const errorMessage = `This command requires ${requiredThings}.`;
```

Comment:
Yes. That's true. A shortcut cannot be required. This code will produce something like this:

<img width="480" alt="bildschirmfoto 2016-06-29 um 16 23 33" src="https://cloud.githubusercontent.com/assets/1606004/16455774/ddedc0c4-3e15-11e6-83d6-d9a4d8f31cac.png">